q2_types/feature_data/_transformer.py

```diff
@@ -262,10 +263,14 @@ def _dnafastaformats_to_metadata(ff):
     return qiime2.Metadata(df)
 
 
-def _series_to_fasta_format(ff, data):
+def _series_to_fasta_format(ff, data, sequence_type="DNA"):
     with ff.open() as f:
         for id_, seq in data.iteritems():
-            sequence = skbio.DNA(seq, metadata={'id': id_})
+            sequence = skbio.Protein(
+                seq, metadata={
+                    'id': id_}) if sequence_type == "protein" else skbio.DNA(
+                seq, metadata={
+                    'id': id_})
```

Comment:
This is a super minor stylistic quibble, but I think this would be a bit easier to read for future readers if the ternary was converted into a tradition `if/else`. My general rule of thumb (in python) is if I can squeeze the ternary into 1 80col line, then I'll do it, anything that requires splitting onto multiple lines usually isn' worth it.